.changeset/cold-moose-exercise.md

```diff
@@ -0,0 +1,44 @@
+---
+"@thirdweb-dev/react-core": minor
```

Comment:
```suggestion
"@thirdweb-dev/react-core": patch
```

these need to be patch versions because of the legacy setup (causes major version bump for dependent packages otherwise)